### PR TITLE
Optionally, delete old backup files

### PR DESCRIPTION
**Pull request by dalenewby**
## Changed

* Reduce the number of backup files to retain to the number specified in the new argument, `--keep`. The argument is optional and defaults to `0` which keeps all backup files.

ITSE-802

### CODE REVIEW

File: youtrack-backup.pl [hunk 1]
```diff
@@ -78,7 +80,8 @@ ()
 my $content_header  = "--header \"Content-Type: application/json\"";
 my $progress_header = "--no-progress-meter";
 
-$delay = 30 if (! defined($delay));
+$delay      = 30 if (! defined($delay));
+$keep_count =  0 if (! defined($keep_count));
```
Comment (mtompset) [inline youtrack-backup.pl]: Please use // instead.
my $a = undef;
my $b = $a // 5;  # $b = 5;

**Comment by mtompset** [comment]
Also, could you clean up the mix of spaces and tabs?

File: youtrack-backup.pl (hunk 1 above)
Comment (mtompset) [reply youtrack-backup.pl]: https://perldoc.perl.org/perlop#Logical-Defined-Or

Comment (dalenewby) [reply youtrack-backup.pl]: Fixed by initializing at time of declaration.

**Review by mtompset — APPROVED** [review]

**Pull request merged**